jams/eval.py

```diff
@@ -32,6 +32,9 @@ def validate_annotation(ann, namespace):
     '''Validate that the annotation has the correct namespace,
```

Comment:
Actually, it's not included in `__all__` or the docs, so it's 100% private and we can do whatever we want without worrying about breaking API.

Shall I change it?